test/regressions/XMLHttpRequest.js

```diff
@@ -0,0 +1,17 @@
+(function() {
```

Comment:
Can you add a comment to inform what is this file and why we need it?
Then link to this PR.